command/loadbot/generator/generator.go

```diff
@@ -3,24 +3,30 @@ package generator
 import (
 	"crypto/ecdsa"
 	"encoding/json"
-	"github.com/0xPolygon/polygon-edge/types"
 	"io/ioutil"
 	"math/big"
+
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/umbracle/go-web3"
+	"github.com/umbracle/go-web3/abi"
 )
 
 type TransactionGenerator interface {
 	GenerateTransaction() (*types.Transaction, error)
 	GetExampleTransaction() (*types.Transaction, error)
 	GetTransactionErrors() []*FailedTxnInfo
 	MarkFailedTxn(failedTxn *FailedTxnInfo)
+	MarkFailedContractTxn(failedContractTxn *FailedContractTxnInfo)
 	SetGasEstimate(gasEstimate uint64)
+	SetContractAddress(types.Address)
```

Comment:
The transaction generator shouldn't need to know about Contract transactions, that was the whole point of separating them out

Reply:
Moved to `ContractTxnGenerator` interface that is used only on `erc` modes